routes/peopleSearch.ts

```diff
@@ -13,7 +13,7 @@ import { RequestWithSystemwidePermissions } from '../transitional';
 
 import { ensureAllLinksInMemory, getAllLinksFromRequest } from '../middleware/business/allLinks';
 
-const lowercaser = require('../middleware/lowercaser.js');
+const lowercaser = require('../middleware/lowercaser');
```

Comment:
General note as we TypeScript... these should become imports once the file exports the method instead... using require is buggy these days in that it hides runtime errors of a small class.